docs/src/main/sphinx/admin/properties-task.rst

```diff
@@ -55,6 +55,18 @@ number of threads.
 Controls staleness of task information, which is used in scheduling. Larger values
 can reduce coordinator CPU load, but may result in suboptimal split scheduling.
 
+``task.max-drivers-per-task``
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+* **Type:** ``integer``
+* **Minimum value:** ``1``
+* **Default Value:** ``2147483647``
+
+Controls the maximum number of drivers a task can run concurrently. Setting this
+value can reduce a single task's ability to use too many drivers, and may improve
+performance when running many concurrent queries. However, this can lead to resource
+waste while running few concurrent queries.
```

Comment:
This sentence sounds a little awkward and I wasn't entirely sure what the last sentence meant. What about:

> "Controls the maximum number of drivers a task runs concurrently. Setting this value reduces the likelihood that a > task uses too many drivers and can improve concurrent query performance. This can lead to resource waste if it runs too few concurrent queries."

Reply:
I think you're right, but what if we combined the two sentences, like:

> Controls the maximum number of drivers a task runs concurrently. Setting this value reduces the likelihood that a task uses too many drivers and can improve concurrent query performance but can lead to resource waste if there are few concurrent queries.

Thoughts?

Reply:
Ok, that sounds good to me - I'll update with your original suggestion. I should also read up on the google dev writing style guide as it sounds like a good reference.